packages/e2e-tests/specs/experiments/block-directory-add.test.js

```diff
@@ -11,7 +11,7 @@ import {
 /**
  * Internal dependencies
  */
-import { enableExperimentalFeatures } from '../../experimental-features';
+import { useExperimentalFeatures } from '../../experimental-features';
```

Comment:
It reads like React hook because of the `use` prefix 🙃

Reply:
Yeah, largely intentional in that the _purpose_ is largely the same, though I grant they are quite different as far as the underlying implementation, and there is some potential for confusion. It feels like the chance of _actual conflict_ is fairly low. I'm open to different naming. In the original Slack conversation, I [mentioned](https://wordpress.slack.com/archives/C02QB2JS7/p1590678951421600?thread_ts=1590677131.416000&cid=C02QB2JS7) `withFooEnabled` as an example, though there's a similar issue there with how we tend to use `with` prefixes for higher-order functions. I also don't think that React should hold exclusive rights to this prefix, since it's quite generic.

🤷

Reply:
It’s totally fine. I like this parallel. I just wanted to emphasize how this concept changed the perception when reading code outside of the React context (it wasn’t intentional to use double meaning here but now I find it funny).

Reply:
> I do like the idea of propagating this sort of pattern for the general requirement of test lifecycle utilities

Related: #22804